src/cursor.rs

```diff
@@ -96,6 +96,11 @@ where
         })
     }
 
+    #[inline]
+    pub fn ptr(&self) -> u16 {
+        self.ptr
+    }
+
```

Comment:
Do we actually want/need to expose this? I assume this was used for debugging?

Reply:
Ah that got mixed in by mistake. It is for another change so I'll remove it for the moment.